README.md

```diff
@@ -62,7 +62,7 @@ class NumberViewHolder(override val containerView: View) :
         RecyclerView.ViewHolder(containerView), LayoutContainer {
     
     @Bind
-    fun bindModel(data: Int, position: Int) {
+    fun bindModel(data: Int) {
```

Comment:
Perhaps we should change the type of this data class. This is a confusing example now, because we are passing in one Int.